src/encoded/static/components/filegallery.js

```diff
@@ -554,12 +612,12 @@ class RawSequencingTable extends React.Component {
             let pairedRepGroups = {};
             let pairedRepKeys = [];
             if (pairedFiles.length > 0) {
-                pairedRepGroups = _(pairedFiles).groupBy(file => (
-                    (file.biological_replicates && file.biological_replicates.length === 1) ?
-                        globals.zeroFill(file.biological_replicates[0]) + ((file.replicate && file.replicate.library) ? file.replicate.library.accession : '')
-                    :
-                        'Z'
-                ));
+                pairedRepGroups = _(pairedFiles).groupBy((file) => {
+                    if (file.biological_replicates && file.biological_replicates.length === 1) {
```

Comment:
How is this an improvement from the previous code?

Reply:
I had initially modified this to do additional things and had to remove the ternary expression. I removed that change, but felt the ternary condition was so long and unwieldy, I preferred using the if.